NLog.Web.AspNetCore.Tests/LayoutRenderers/AspNetRequestPostedBodyTests.cs

```diff
@@ -59,8 +59,10 @@ private static void SetBodyStream(HttpContextBase httpContext, Stream stream)
         {
 #if ASP_NET_CORE
             httpContext.Request.Body.Returns(stream);
+            httpContext.Request.ContentLength.Returns((int)(stream?.Length ?? 0));
 #else
             httpContext.Request.InputStream.Returns(stream);
+            httpContext.Request.ContentLength.Returns((int)(stream?.Length ?? 0));
```

Comment:
Could this setup after the #if? (Duplicate lines)

Reply:
Fixed